core/unit_test/TestReducers.hpp

```diff
@@ -1012,19 +1233,44 @@ struct TestReducers {
 #if !defined(KOKKOS_ENABLE_OPENACC)
     // FIXME_OPENACC - OpenACC (V3.3) does not support custom reductions.
     test_minloc(10003);
+#if defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC misaligned memory
+#if defined(KOKKOS_ENABLE_CUDA)
+    if (!std::is_same_v<ExecSpace, Kokkos::Cuda>)
+#endif
+    // FIXME_OPENMPTARGET requires custom reductions.
+#if !defined(KOKKOS_ENABLE_OPENMPTARGET)
+      test_minloc_2d(100);
+#endif
+#endif
 #endif
     test_max(10007);
 #if !defined(KOKKOS_ENABLE_OPENACC)
     // FIXME_OPENACC - OpenACC (V3.3) does not support custom reductions.
     test_maxloc(10007);
+#if !defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC misaligned memory
+#if defined(KOKKOS_ENABLE_CUDA)
+    if (!std::is_same_v<ExecSpace, Kokkos::Cuda>)
 #endif
-#if defined(KOKKOS_ENABLE_OPENMPTARGET) && defined(KOKKOS_COMPILER_CLANG) && \
-    (KOKKOS_COMPILER_CLANG < 1300)
-    // FIXME_OPENMPTARGET - The minmaxloc test fails llvm <= 13 version.
-#else
+// FIXME_OPENMPTARGET requires custom reductions.
+#if !defined(KOKKOS_ENABLE_OPENMPTARGET)
+      test_maxloc_2d(100);
+#endif
+#endif
+#endif
+// FIXME_OPENACC - OpenACC (V3.3) does not support custom reductions.
 #if !defined(KOKKOS_ENABLE_OPENACC)
-    // FIXME_OPENACC - OpenACC (V3.3) does not support custom reductions.
+// FIXME_OPENMPTARGET - The minmaxloc test fails llvm < 13 version,
+// the minmaxloc_2d test requires custom reductions.
+#if defined(KOKKOS_ENABLE_OPENMPTARGET)
+#if defined(KOKKOS_COMPILER_CLANG) && (KOKKOS_COMPILER_CLANG >= 1300)
+    test_minmaxloc(10007);
+#else
+    if (!std::is_same_v<ExecSpace, Kokkos::Experimental::OpenMPTarget>)
```

Comment:
should that be `if constexpr`?  @rgayatri23

Reply:
what is our minimum requirement?

Reply:
Yeah `if constexpr` would work there. We don't have a minimum requirement for clang version but I would say not to try anything with less than 14 for all enabled unit tests to pass

Reply:
Should we then formalize using clang-14 as the minimum in another pull request?

Reply:
Yes sure. I can do that.

Reply:
the question was "does it need to be constexpr" meaning was it a compile or runtime error.

Reply:
I do not remember. But if we restrict OpenMPTarget backend to use minimum clang-14, then we can avoid that condition